jbpm-designer-client/src/main/resources/org/jbpm/designer/public/simulation/pathimage.jsp

```diff
@@ -8,6 +8,18 @@
 <script>
     function showPathImage() {
         var pathSVG = parent.getPathSVG();
+        var isChrome = /Chrome/.test(navigator.userAgent) && /Google Inc/.test(navigator.vendor);
```

Comment:
Is it possible to add other chromium based browsers at least for community version?